### PR TITLE
Features/theme basic simple pager

### PR DESCRIPTION
**Pull request by azkurban**
Simple Pager without Pager Target feature

### CODE REVIEW

File: src/theme/basic/index.scss [hunk 1]
```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
...
+    }
+
+    .q-grid-pager {
+        font-family: Roboto, "Helvetica Neue", sans-serif;
+        padding: 11px 0;
+        height: 24px;
+        font-size: 12px;
+        color: $pager-primary;
+
+        .pager-select {
```
Comment (klumba12) [inline src/theme/basic/index.scss]: all classes should start with q-grid - q-grid-pager-select in this case

File: src/theme/basic/index.scss [hunk 2]
```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
...
+                height: 24px;
+                width: 34px;
+                margin: 0 0 0 12px;
+            }
+            select:focus {
+                outline: none;
+            }
+        }
+
+        .q-grid-page {
```
Comment (klumba12) [inline src/theme/basic/index.scss]: q-grid-pager-page?

File: src/theme/basic/index.scss [hunk 3]
```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
...
+    .q-grid-pager button {
+        background: transparent;
+        border: none;
+        font-size: 20px;
+        width: 24px;
+        height: 24px;
+        line-height: 24px;
+        padding-top: 2px;
+
+        &.q-grid-prev-page {
```
Comment (klumba12) [inline src/theme/basic/index.scss]: lets make q-grid-pager prefix

File: src/theme/basic/index.scss (hunk 2 above)
Comment (azkurban) [reply src/theme/basic/index.scss]: That class has been copied from the corresponding Material template. So, please, keep in mind that we have the same issue there as well.

But I will fix this anyway.

File: src/theme/basic/index.scss (hunk 3 above)
Comment (azkurban) [reply src/theme/basic/index.scss]: The same caveat as I described in my previous comment.

File: src/theme/basic/index.scss (hunk 1 above)
Comment (azkurban) [reply src/theme/basic/index.scss]: done in my last commit

File: src/theme/basic/index.scss (hunk 2 above)
Comment (azkurban) [reply src/theme/basic/index.scss]: done in my last commit

File: src/theme/basic/index.scss (hunk 3 above)
Comment (azkurban) [reply src/theme/basic/index.scss]: done in my last commit.

**Pull request merged**